hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacilityTest.java

```diff
@@ -83,7 +88,7 @@ void testMethodsWithInvalidArguments() {
         final var txBody = simpleCryptoTransfer().body();
 
         // then
-        assertThatThrownBy(() -> new SystemFileUpdateFacility(null)).isInstanceOf(NullPointerException.class);
+        assertThatThrownBy(() -> new SystemFileUpdateFacility(null, null)).isInstanceOf(NullPointerException.class);
```

Comment:
We should have two checks here, one for each parameter.

Reply:
Will fix it in https://github.com/hashgraph/hedera-services/pull/8147